client/src/Components/ShippingInformation.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+
+class ShippingInformation extends React.Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      shippingTime: [],
+      exclusions: [],
+      returnPolicy : [],
+      returnOverview : [],
+      giftWrapping: [],
+      getShippingCost: false,
+      shippingCost: [],
+      enteredShippingInfo: false,
+      zipCode : [],
+      shippingFrom: [],
+      shippingTo: [],
+      shippingToCountry: [],
+      deliveryDate : []
+    }
+    this.getShippingInformation = this.getShippingInformation.bind(this);
+    this.getReturnPolicy = this.getReturnPolicy.bind(this);
+    this.getShippingInfo = this.getShippingInfo.bind(this);
+    this.showShipping = this.showShipping.bind(this);
+    this.shippingCost = this.shippingCost.bind(this);
+    this.shippingToCountry = this.shippingToCountry.bind(this);    
+  }
+
+
+  componentDidMount(){
+    this.getShippingInformation()
+    this.getReturnPolicy()      
+  }
+  getShippingInformation(){
+    let manufacturingTime;
+    let shippingFrom = this.props.data[0]['shippingFrom']
+    let shippingTo = this.props.data[0]['shippingTo']
+    let deliveryDate = this.props.data[0]['deliveryDate']
+    manufacturingTime = this.props.data[0]['manufacturingTime'];
+    this.setState({
+      shippingTime: manufacturingTime,
+      shippingFrom: shippingFrom,
+      shippingTo: shippingTo,
+      deliveryDate : deliveryDate
+    })
+  }
+
+  getReturnPolicy(){
+    var returnable;
```

Comment:
Always set declared variables to `null` if you are not giving a value until later